client/my-sites/plugins/disconnect-jetpack/disconnect-jetpack-button.jsx

```diff
@@ -24,19 +25,18 @@ class DisconnectJetpackButton extends Component {
 
 	handleClick = event => {
 		event.preventDefault();
-		const { isMock, recordGoogleEvent: recordGAEvent } = this.props;
+		const { isMock } = this.props;
 
 		if ( isMock ) {
 			return;
 		}
 		this.setState( { dialogVisible: true } );
-		recordGAEvent( 'Jetpack', 'Clicked To Open Disconnect Jetpack Dialog' );
+		recordGoogleEvent( 'Jetpack', 'Clicked To Open Disconnect Jetpack Dialog' );
```

Comment:
Instead of directly using this function let's inject it via `connect()` as it's designed to be - note also that it doesn't actually do anything but return a Redux action, so right here it shouldn't be actually recording anything.

```js
const mapDispatchToProps = ( {
	recordGoogleEvent,
} )
```

In the `connect()`ed case the results of calling that function get passed into `dispatch()` and then the [analytics middleware](https://github.com/Automattic/wp-calypso/tree/master/client/state/analytics) will pick it up and actually ship off the metric